operations/node_management/Rebuild_NCNs/Re-add_Storage_Node_to_Ceph.md

```diff
@@ -6,21 +6,14 @@ Use the following procedure to re-add a Ceph node to the Ceph cluster.
 
 ## Add Join Script
 
-1. Copy and paste the below script into `/srv/cray/scripts/common/join_ceph_cluster.sh`.
-
-   **`NOTE`** This script may also available in the `/usr/share/doc/csm/scripts` directory where the latest ***`docs-csm`*** RPM is installed. If so, it can be copied from that node to the new storage node being rebuilt and skip to step 2.
+1. (`ncn-s#`) Copy and paste the below script into `/srv/cray/scripts/common/join_ceph_cluster.sh` **on the node being rebuilt**. If there is an existing script in that location, then replace it.
```

Comment:
Same comment https://github.com/Cray-HPE/docs-csm/pull/3250/files#r1099172475

If these scripts need to live in the image, we can add them into node-images.